setup.py

```diff
@@ -29,7 +29,7 @@ def process_options():
             Is this a release build (True) or a local development build (False)
         'openmp': bool
             Should we build our OpenMP extensions and attempt to link in OpenMP
-            libraries?
+            libraries? (Not supported in this version.)
```

Comment:
We should make a TODO to either remove openmp support or add it back (maybe there is already an open issue?)

Reply:
The hope it to add it back, it's on multiple to-dos (https://github.com/qutip/qutip/projects/6), but no issue has been opened.
Aton said he would take a look at it.